include/tesseract/baseapi.h

```diff
@@ -564,7 +564,7 @@ class TESS_API TessBaseAPI {
    * page_number is 0-based but will appear in the output as 1-based.
    * Returned string must be freed with the delete [] operator.
    */
-  char *GetTSVText(int page_number);
```

Comment:
This is an API change. It requires a new major version (Tesseract 6.0.0) and changes in other software like for example [tesserocr](https://github.com/sirfz/tesserocr).

Therefore we cannot simply merge this pull request.

Reply:
Understood. If you have a suggestion how to provide this functionality without modifying the API, I could steer the PR in that direction.

Reply:
Use overload
```
char *GetTSVText(int page_number, bool lang_info=false);
to
char *GetTSVText(int page_number, bool lang_info);
```

Reply:
I have pushed the overload approach, it does not break the API now.